example_files/example_project_with_ghe.json

```diff
@@ -0,0 +1,1013 @@
+                "geometryType": "Rectangle",
+                "name": "New District System_2",
+                "type": "District System",
+                "district_system_type": "Ground Heat Exchanger",
+                "footprint_area": 104693,
+                "footprint_perimeter": 1332,
+                "floor_area": 104693
+            },
+            "geometry": {
+                "type": "Polygon",
+                "coordinates": [
+                    [
+                        [
+                            -78.85273990319104,
+                            42.81588851783172
+                        ],
+                        [
+                            -78.85224049713851,
+                            42.81481606407624
+                        ],
+                        [
+                            -78.85134704694869,
+                            42.81503993895839
+                        ],
+                        [
+                            -78.85184645300123,
+                            42.81611238883147
+                        ],
+                        [
+                            -78.85273990319104,
+                            42.81588851783172
+                        ]
+                    ]
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "id": "d18cf419-a301-4e1e-9694-613cff741e70",
+                "type": "ThermalJunction",
+                "DSId": "dd69549c-ecfc-4245-96dc-5b6127f34f46",
+                "is_ghe_start_loop": true,
+                "junction_type": "DES",
+                "connection_type": "Series"
+            },
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -78.8527736847739,
+                    42.81787596282345
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "id": "6cf3a36e-705e-4b98-83ef-7f97d1981583",
+                "type": "ThermalJunction",
+                "buildingId": "10",
+                "junction_type": "DES",
+                "connection_type": "Series"
+            },
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -78.85180303988922,
+                    42.81811676138841
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "type": "ThermalConnector",
+                "id": "1c63c10c-2315-4353-b3e9-3127651e6f45",
+                "startJunctionId": "d18cf419-a301-4e1e-9694-613cff741e70",
+                "endJunctionId": "6cf3a36e-705e-4b98-83ef-7f97d1981583",
+                "startFeatureId": "dd69549c-ecfc-4245-96dc-5b6127f34f46",
+                "endFeatureId": "10",
+                "total_length": 274,
+                "connector_type": "OnePipe",
```

Comment:
properties not required for thermal connector and thermal junction can be removed once geojson gem is released

Reply:
TODO These still need to be removed when this branch is updated with the geojson gem release